inference-engine/src/low_precision_transformations/src/subgraph.cpp

```diff
@@ -82,7 +50,7 @@ bool Subgraph::fillSubgraphForQuantization(
                 if (fakeQuantizeChild != nullptr) {
                     //
                 } else {
-                    if (layerTransformationsManager->isPrecisionPreserved(child) && isQuantizationPerChannel(child)) {
```

Comment:
Check `Concat` `axes`, while `FakeQuantize` operation supports per channel & per tensor quantization only: for per channel quantization it has to be 1 and doesn't matter for per tensor quantization.

Reply:
At the moment ConcatTransformation support only concat by channel (`axis = 1`). I think, we don't need to change this behaviour.